### DISCUSSION TITLE
Suggest a method to support querying both quantity and data at the same time

### DISCUSSION THREAD

**Discussion by lnool** [prisma/prisma#15070] [Q&A]
Suggest a method to support querying both quantity and data at the same time
建议支持同时查询数量和数据的方法

**Comment by janpio** [comment]
https://translate.google.com/?sl=auto&tl=en&text=%E5%BB%BA%E8%AE%AE%E6%94%AF%E6%8C%81%E5%90%8C%E6%97%B6%E6%9F%A5%E8%AF%A2%E6%95%B0%E9%87%8F%E5%92%8C%E6%95%B0%E6%8D%AE%E7%9A%84%E6%96%B9%E6%B3%95&op=translate

**Comment by pantharshit00** [comment]
Hello @Inool, our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 

You can the quantity and data at same time using the following:
```ts
  const data = await prisma.yourmodel.findMany({
    include:{
      _count: true
    },
    where:{
      // conditions here
    }
  });
```

**Reply by pantharshit00** [reply]
Hello @lnool,

I just want to make sure if you got a chance to try this out. Let me know if this issue if still relevant to you.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
